### DISCUSSION TITLE
Velocity Support

### DISCUSSION THREAD

**Discussion by Strahilchu** [Zrips/CMI#7727] [Feature Requests]
CMI should support Velocity.
A lot of servers are switching from Bungee and this will only become a more and more demanded feature with time.

It is inevitable to see people ask for this, or avoid CMI when they see it does not support it!

**Comment by CloudeLecaw** [comment]
Doubled discussion.
May you want to discuss here:
https://github.com/Zrips/CMI/discussions/2515

**Comment by CloudeLecaw** [comment]
Can be closed because discussion allready exist

**Comment by mrfloris** [comment]
https://www.spigotmc.org/resources/cmi-298-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/update?update=521168

There is a mention in the changelog for velocity support in today's CMI release.
You can get the free cmiv jar file here: https://www.zrips.net/cmiv/
